utils/parametric/_library_client.py

```diff
@@ -729,6 +730,7 @@ def get_tracer_config(self) -> Dict[str, Optional[str]]:
             "dd_trace_sample_ignore_parent": config_dict.get("dd_trace_sample_ignore_parent", None),
             "dd_env": config_dict.get("dd_env", None),
             "dd_version": config_dict.get("dd_version", None),
+            "dd_trace_agent_url": config_dict.get("dd_trace_agent_url", None),
```

Comment:
Should we also modify the protos and grpc endpoints in this PR?

Even if this test isn't enabled for all libraries, we should write this test in a way where it can easily be enabled.

Reply:
What needs to be modified? for the existing proto:

rpc GetTraceConfig(GetTraceConfigArgs) returns (GetTraceConfigReturn) {}

GetTraceConfigReturn just returns a map<string, string> config 

so to enabled the test on the grpc end, parametrics apps just need to be updated to pass in the dd_trace_agent_url key in the map

Reply:
You're right we don't need to modify protos for this change.